server/server/Controllers/AdminController.cs

```diff
@@ -25,38 +25,16 @@ public AdminController(ApplicationContext context)
             db = new UnitOfWork(context);
         }
 
-        static void SendEmail(string from, string to, string name, string cakeDescription, string login, string password)
-        {       
-            MimeMessage message = new MimeMessage();
-            message.From.Add(new MailboxAddress("MasterCake", from));
-            message.To.Add(new MailboxAddress("", to));
-            message.Subject = "New cake in MasterCake";
-            message.Body = new BodyBuilder() { HtmlBody = 
-                "<div " +
-                    "<div style=\"padding-bottom: 5%;\"><em>Dear " + name  + ",</em></div>" +
-                    "<b style=\"color: purple;\">We have a new cake in our catalog especially for you!</b>" +
-                    "<div style=\"padding: 10% 0;\"><em>" + cakeDescription + "</em></div>" + 
-                    "<b>Hurry to order!</b>" +
-                    "<div style=\"padding-top: 20%;\"><em>With love, your MasterCake)</em></div>" +
-                "</div>" }.ToMessageBody(); 
-
-            using (MailKit.Net.Smtp.SmtpClient client = new MailKit.Net.Smtp.SmtpClient())
-            {
-                client.Connect("smtp.mail.ru", 25, SecureSocketOptions.StartTls);
-                client.Authenticate(login, password);
-                client.Send(message);
-
-                client.Disconnect(true);
-            }
-        }
+        
 
         [HttpPost]
         [Authorize]
         public IActionResult Post(AddedCake cake)
         {
             char separator = System.Globalization.CultureInfo.CurrentCulture.NumberFormat.CurrencyDecimalSeparator[0];
             string stringPrice = cake.Price.Replace('.', separator);
-            decimal price = Convert.ToDecimal(stringPrice);
+            decimal price = 0.0M;
+            decimal.TryParse(stringPrice, out price);
```

Comment:
What if price contains invalid string, then 0.0 will be saved to db. Better to throw exception